rest-api/src/main/resources/definitions/assessment.yml

```diff
@@ -118,6 +118,10 @@ properties:
     additionalMetadata:
         type: object
         description: Additional metadata fields, expressed as a key-value map.
+    phase:
+        $ref: ./enums/assessment_phase.yml
+        x-nullable: false
+        description: One of an enumerated list of states an assessment can be in.
```

Comment:
In our Code Conventions doc (https://sagebionetworks.jira.com/wiki/spaces/BRIDGE/pages/74219580/Code+Conventions#CodeConventions-DocumentationoffieldsinourSwaggerAPIdefinition), x-nullable doesn't actually do anything, and we're no longer adding this to our JavaSDK. For required fields, we're now adding it to the "required" list at the top of the file.

Reply:
I'm pretty sure the kotlin code generation we use for BridgeClientKMM pays attention to the x-nullable field in the swagger file.

Reply:
Interesting. I didn't know that. We should verify that and update the Code Conventions accordingly.